### DISCUSSION TITLE
Allow converting Discussion to Issue

### DISCUSSION THREAD

**Discussion by kzu** [community/community#3297] [Discussions] [answered]
Discussions are great to flesh out potential new features proposed by users (i.e. https://github.com/devlooped/ThisAssembly/discussions/34). Once it's been agreed it would be a good issue/feature to work on, in some cases it might be handy to just be able to turn the discussion into an Issue directly and label it appropriately.

**Comment by TG-Techie** [comment]
It is a feature I tried to find/use after the first successful discussion had

When you say "into an Issue directly" are you proposing that the comments / etc would be lifted and transferred into comments on the issue as if they were typed there?

I think having a section where the user turning the discussion into an issue can type a small summary and list of actionables would be useful.

**Comment by bartr** [comment]
If you have permissions, you can do this by clicking the `...` on the discussion. It "references" the discussion in an issue vs. "converting".  We use it all the time and it's working great.

![image](https://user-images.githubusercontent.com/1355059/110902067-e0b69980-82ca-11eb-9b37-ae5c31a50514.png)

**Comment by riverar** [comment]
We find that customers aren't immediately sure if a particular topic is a discussion or issue and create a discussion in error. We then find ourselves needing to take action on that particular item, but without the same issue resolution tools available to us. It would be great to convert a discussion into an issue, maintaining the original author.

`Reference in new issue` is a decent workaround for now though.

**Comment by kernc** [comment]
> just be able to turn the discussion into an Issue directly and label it appropriately

Just that. In most cases, a discussion turns out to be an issue very _quickly_, and there's plenty of discussion in issues anyway.

**Comment by Torxed** [comment]
I'm absolutley in favor of this, for instance in cases like this: https://github.com/archlinux/archinstall/discussions/168

Where a conversation later turned out to be a work item/issue.
But now It's kind of stuck in a discussion forum forever.. Which doesn't make sense because it doesn't really have an "answer".

**Comment by driesvints** [comment, marked as answer]
Seems to be available now! 🎉 

![Screen Shot 2021-07-01 at 09 25 41](https://user-images.githubusercontent.com/594614/124083578-526b4c80-da4e-11eb-826e-0c0a848b7bb1.jpg)

**Reply by Torxed** [reply]
That's great, sadly it doesn't move the discussion to an issue. It saves the discussion rather than the counterpart moving an issue to discussion :)
